shared/src/util/enzymeSnapshotModifiers.ts

```diff
@@ -2,16 +2,51 @@
 // These are used in share/dev/enzymeSerializer.js
 
 import { Json } from 'enzyme-to-json'
+import * as H from 'history'
+import { isPlainObject } from 'lodash'
+import { Observable, Subject } from 'rxjs'
+
+function maskProps(props: Json['props'], key = '', seenProperties = new Set<unknown>()): unknown {
```

Comment:
Little improvement to avoid mutation :)
```suggestion
function maskProps(props: Json['props'], key = '', seenProperties: ReadonlySet<unknown> = new Set()): unknown {
```

Reply:
Oh, I didn't know that exists!